README.md

```diff
@@ -25,14 +25,53 @@ Let's create a simple app that integrates PSPDFKit and uses the Flutter pspdfkit
 4. Open `myapp/android/local.properties` and specify the following properties
 
 ```local.properties
-ndk.dir=/path/to/your/Android/sdk/ndk-bundle
 sdk.dir=/path/to/your/Android/sdk
 flutter.sdk=/path/to/your/flutter/sdk
 pspdfkit.password=YOUR_PASSWORD_GOES_HERE
 flutter.buildMode=debug
 ```
 
-5. Open `myapp/android/app/build.gradle` and modify line 27 from `minSdkVersion 16` to `minSdkVersion 19`.
+5. Open `myapp/android/app/build.gradle` and modify `compileSdkVersion` from `27` to `28`, `minSdkVersion` from `16` to `19`, `targetSdkVersion` from `27` to `28` and add compile options to enable desugaring 
```

Comment:
I would also mention that we require `androidGradlePluginVersion` to be `3.2.1`.